api/provider/aws/load_balancer_read.go

```diff
@@ -80,10 +76,14 @@ func (l *LoadBalancerProvider) describeLoadBalancer(loadBalancerName string) (*e
 	return output.LoadBalancerDescriptions[0], nil
 }
 
-func (l *LoadBalancerProvider) populateModelTags(loadBalancerID string, model *models.LoadBalancer) error {
+func (l *LoadBalancerProvider) newModel(loadBalancerID string) (*models.LoadBalancer, error) {
```

Comment:
These functions are super specific to their use case in this particular CRUD operation, but have such a general name. In the `Create()` portion, we'd also need to call a `newModel` function, wouldn't we?